### ISSUE TITLE
windows 10, python 3.5, unicodeencodeerror

### ISSUE THREAD

**Issue by denfromufa** [chrisjsewell/ipypublish#46]
```
C:\Python\Python35_64b\Scripts>python nbpublish.py -f nb_ipypublish_all -o C:\Users\denis\Dropbox\nau_course\lecture-02-pydata C:\Users\denis\Dropbox\nau_course\lecture-02-pydata\WhirlwindTourOfPython
INFO:main:started ipypublish at Mon Sep  4 11:57:50 2017
INFO:main:logging to: C:\Users\denis\Dropbox\nau_course\lecture-02-pydata\WhirlwindTourOfPython.nbpub.log
INFO:main:running for ipynb(s) at: C:\Users\denis\Dropbox\nau_course\lecture-02-pydata\WhirlwindTourOfPython
INFO:main:with conversion: nb_ipypublish_all
INFO:nbmerge:Merging all notebooks in directory
INFO:main:getting output format from exporter plugin
INFO:nbexport:running nbconvert
WARNING:configurable:Config option `filters` not recognized by `MyExporter`.
INFO:latex_doc_defaults:adding ipub defaults to notebook
INFO:main:outputting converted file to: C:\Users\denis\Dropbox\nau_course\lecture-02-pydata\WhirlwindTourOfPython.ipynb
Traceback (most recent call last):
  File "nbpublish.py", line 142, in <module>
    nbpublish(filepath, **options)
  File "nbpublish.py", line 66, in nbpublish
    create_pdf=create_pdf, pdf_in_temp=pdf_in_temp, pdf_debug=pdf_debug)
  File "C:\Python\Python35_64b\lib\site-packages\ipypublish\main.py", line 129, in publish
    fh.write(body)
  File "C:\Python\Python35_64b\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2010' in position 7108: character maps to <undefined>

C:\Python\Python35_64b\Scripts>
```

**Comment by chrisjsewell** [comment]
Hey thanks for the feedback, I think I'd actually fixed this already (395c51b) but hadn't released a new version. If you update your ipypublish version to 0.6.4 (`pip install --upgrade ipypublish` if you didn't know) and let me know how that goes :)

Unfortunately [Travis](https://travis-ci.org/chrisjsewell/ipypublish/builds) doesn't let me run the tests directly in the Windows environment, so hopefully there is no unforeseen issues. Also its probably advisable to use python 3.6 (although the tests are passing for Linux in 3.5 so should be ok).

**Comment by chrisjsewell** [comment]
I'm going to close this, as there has been no response, but feel free to reopen if there are any other issues
